bin/cheritest/Makefile

```diff
@@ -72,15 +75,26 @@ MAN=
 
 .if ${PROG:Mcheritest*}
 NEED_CHERI=	hybrid
+.if ${MACHINE_ABI:Mpurecap}
+NEED_COMPAT=	64
+.include <bsd.compat.mk>
```

Comment:
It would be nice if this were a no-op for !purecap...

Reply:
Good point, I'll fix that. This also prompted me to create #502 to better sync mips and riscv